edx_proctoring/backends/rest.py

```diff
@@ -200,6 +201,20 @@ def get_instructor_url(self, course_id, user, exam_id=None, attempt_id=None):
         log.debug('Created instructor url for %r %r %r', course_id, exam_id, attempt_id)
         return url
 
+    def _get_language_headers(self):
+        """
+        Makes the Accept-Language header based on the currently active language
+        """
+        from django.conf import settings
+        from django.utils.translation import get_language
+
+        headers = {}
+        current_lang = get_language()
+        default_lang = settings.LANGUAGE_CODE
+        if current_lang and current_lang != default_lang:
```

Comment:
Does it hurt to send the header even if the current language is the default?  Being explicit with a third-party service would be better, no?

Reply:
It'll be sent all the time now.

Reply:
Cool.